docs/resources/security_policy.md

```diff
@@ -119,6 +120,7 @@ Optional:
 Optional:
 
 - **cvss_range** (Block List, Max: 1) Nested block describing a CVS score range to be impacted. Defined below. (see [below for nested schema](#nestedblock--rule--criteria--cvss_range))
+- **fix_version_dependant** (Boolean) Issues that do not have a fixed version are not generated until a fixed version is available.
```

Comment:
Include default value?

Reply:
Auto-generated. Do we add default values in this case? i don't like the idea to fix manually whatever was generated

Reply:
When we will generate docs the next time, the manual change will be removed

Reply:
See comment below. Add default value to attribute's description.

Reply:
Added to the description